specification/src/main/asciidoc/coreprofile/CoreProfileDefinition.adoc

```diff
@@ -0,0 +1,30 @@
+== Core Profile Definition
+
+This chapter defines the contents of the Jakarta™ EE 10.0 Core Profile.
+
+[[required_components]]
+=== Required Components
+
+The following technologies are required components of the Core Profile:
+
+* Jakarta RESTful Web Services 4.0
+* Jakarta JSON Processing 2.1
+* Jakarta JSON Binding 2.1
+* Jakarta Annotations 2.0
+* Jakarta Enterprise Beans Lite 4.0
```

Comment:
@starksm64  Jakarta Enterprise Beans Lite 4.0-> Jakarta Contexts and Dependency Injection Lite 4.0?